packages/protocol/graphics.ts

```diff
@@ -274,7 +283,8 @@ export async function repaint(force = false) {
     }
   }
 
-  if (!rv || pauseId !== ThreadFront.currentPause.pauseId) {
+  // if (!rv || pauseId !== ThreadFront.currentPause.pauseId) {
+  if (!rv) {
```

Comment:
@hbenl - Looks like you originally added this condition in #4396. In this refactor, I want to explicitly allow fetching and repainting the screen outside of the current pause. However, I can also rework this method (`repaintAtPause`) to not be used by the original `repaint` method so that we can retain this condition. I wasn't sure why it was there so I wanted to ask.

Reply:
@hbenl - Lemme know if you recall why we had this condition originally. Thanks!

Reply:
The condition is there to avoid painting a screenshot for a pause that the user has already switched away from.
I think the `repaintAtPause()` function should not paint the screenshot that it fetches - we should always show the screenshot for the current pause. Instead it should only fetch it (and be renamed to `fetchRepaintForPause()` or something) and the `paintGraphics()` call should be moved to the `repaint()` method together with the `pauseId !== ThreadFront.currentPause.pauseId` condition.

Reply:
We need the ability to paint the screenshot for a different pause -- that's the feature we're implementing here to preview a test step without jumping to that point -- but your explanation for that condition makes sense so I'll refactor with that in mind.